package.json

```diff
@@ -100,5 +102,6 @@
     "setupFilesAfterEnv": [
       "./test/jest.setup.js"
     ]
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha1.ac34549e6aa8e7ead463a7407e1c7390f61a6610"
```

Comment:
we don't use yarn, and this shouldn't be here. Corepack has been removed from node